src/proof/JsonWebSignatureProof.ts

```diff
@@ -0,0 +1,166 @@
+// TODO: Remove this disable as part of IDCOM-2356
+/* eslint-disable */
+import {VerifiableCredential} from "../vc/VerifiableCredential";
+import cre from "@transmute/credentials-context";
+import sec from "@transmute/security-context";
+import didContext from "@transmute/did-context";
+import {Ed25519VerificationKey2018} from "@transmute/ed25519-key-pair";
+import {
+    JsonWebKey,
+    JsonWebSignature,
+    JsonWebKey2020,
+} from "@transmute/json-web-signature";
+import {verifiable} from "@transmute/vc.js";
+import {IDiDResolver} from "../lib/resolver";
+import Proof from "./Proof";
+import {VerificationMethod} from "did-resolver";
+
+// The credential context that includes additional properties
+const credentialContext = {
+    '@context': {
+        '@version': 1.1,
+        '@protected': true,
+        id: '@id',
+        type: '@type',
+        xsd: 'http://www.w3.org/2001/XMLSchema#',
+        IdentityCredential: {
+            '@id': 'https://www.identity.com/credentials/v1#identityCredential',
+            '@context': {
+                '@version': 1.1,
+                '@protected': true,
+                identifier: {
+                    '@id': 'cvc:identifier',
+                    '@type': 'xsd:string',
+                },
+                version: {
+                    '@id': 'cvc:version',
+                    '@type': 'xsd:decimal',
+                },
+            },
+        },
+    },
+};
+
+const updateCredentialContext = (context: any) => {
+    context["@context"].VerifiableCredential["@context"].credentialSubject = {
+        '@id': 'cred:credentialSubject',
+        '@type': '@json'
+    };
+
+    return context;
+}
+
+export const contexts: any = {
+    [cre.constants.CREDENTIALS_CONTEXT_V1_URL]: updateCredentialContext(cre.contexts.get(
+        cre.constants.CREDENTIALS_CONTEXT_V1_URL
+    )),
+    [didContext.constants.DID_CONTEXT_V1_URL]: didContext.contexts.get(
+        didContext.constants.DID_CONTEXT_V1_URL
+    ),
+    [sec.constants.JSON_WEB_SIGNATURE_2020_V1_URL]: sec.contexts.get(
+        sec.constants.JSON_WEB_SIGNATURE_2020_V1_URL
+    ),
+    [sec.constants.BLS12381_2020_V1_URL]: sec.contexts.get(
```

Comment:
Not all of these contexts are needed... will figure out the exact one's and remove along with the ECDSA ticket